docs/general/installation/truenas.md

```diff
@@ -0,0 +1,246 @@
+For example, the local **Host Path** /mnt/tank/video/movies could be assigned the **Mount Path** /media/movies.
+
+<details>
+  <summary>Additional Storage Fields</summary>
+
+  <table>
+    <thead>
+      <tr>
+        <th>Type</th>
+        <th>Field</th>
+        <th>Description</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>All</td>
+        <td>Mount Path</td>
+        <td>The virtual path to mount the storage within the container.</td>
+      </tr>
+      <tr>
+        <td>Host Path</td>
+        <td>Host Path</td>
+        <td>The local path to an existing dataset on the System.</td>
+      </tr>
+      <tr>
+        <td>ixVolume</td>
+        <td>Dataset Name</td>
+        <td>The name for the dataset the system creates.</td>
+      </tr>
+      <tr>
+        <td>SMB Share</td>
+        <td>Server</td>
+        <td>The server for the SMB share.</td>
+      </tr>
+      <tr>
+        <td>SMB Share</td>
+        <td>Share</td>
+        <td>The name of the share.</td>
+      </tr>
+      <tr>
+        <td>SMB Share</td>
+        <td>Domain (Optional)</td>
+        <td>The domain for the SMB share.</td>
+      </tr>
+      <tr>
+        <td>SMB Share</td>
+        <td>Username</td>
+        <td>The user name used to access the SMB share.</td>
+      </tr>
+      <tr>
+        <td>SMB Share</td>
+        <td>Password</td>
+        <td>The password for the SMB share user.</td>
+      </tr>
+      <tr>
+        <td>SMB Share</td>
+        <td>Size (in Gi)</td>
+        <td>The quota size for the share volume. You can edit the size after deploying the application if you need to increase the storage volume capacity for the share.</td>
+      </tr>
+    </tbody>
+  </table>
+</details>
+
+### Resource Configuration Settings
+
+Accept the default values in **Resources Configuration** or enter new CPU and memory values
+By default, this application is limited to use no more than 4 CPU cores and 8 gibibytes available memory.
+
+![Resource Limits](/images/docs/install-truenas-12.png)
+
+To customize the CPU and memory allocated to the container Jellyfin uses, enter new CPU values as a plain integer value followed by the suffix m (milli).
+Default is 4000m.
+
+Accept the default value 8Gi allocated memory or enter a new limit in bytes.
+Enter a plain integer followed by the measurement suffix, for example 129M or 123Mi.
+
+Systems with compatible GPU(s) for hardware acceleration display devices in **GPU Configuration**.
+Use the **GPU Resource** dropdown menu(s) to configure device allocation.
+
+To isolate a GPU, go to the **System Settings > Advanced** screen and click **Configure** on the **Isolated GPU Device(s)** widget to open the **Isolate GPU PCI’s Ids** screen. Select the GPU device ID to isolate from the dropdown list and click **Save**.
+You must have at least two GPUs in your system; one allocated to the host system for system functions and the other available for use by either a VM or applications.
```

Comment:
Is having a single SR-IOV capable GPU acceptable for this? If so it should be noted, as many Intel Integrated Graphics devices have this capability.

Reply:
I'm not entirely clear on the question here, but my understanding is a dedicated GPU is required for system processes. An additional SR-IOV capable GPU could be split across multiple apps, but not between system tasks and app processing. Does that make sense?

Reply:
The question here is, if I only have a single GPU on the system that is capable of SR-IOV, can I use it for both the host and apps?

Reply:
It appears I was misinformed here. On further investigation the limitation is VM specific, so a single GPU cannot be split between host and VM system tasks, but a single GPU should work for system and apps. I'm going to suggest simply removing lines 243 and 244.